src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -2389,23 +2464,18 @@
 short-summary: Update a VMSS.
 examples:
   - name: Update a VMSS. (autogenerated)
-    text: az vmss update --name MyScaleSet --resource-group MyResourceGroup --set virtualMachineProfile.storageProfile.imageReference.version=16.04.201801090
+    text: |
+        az vmss update --name MyScaleSet --resource-group MyResourceGroup --set virtualMachineProfile.storageProfile.imageReference.version=16.04.201801090
     crafted: true
-  - name: Update a VMSS' license type for Azure Hybrid Benefit.
-    text: az vmss update --name MyScaleSet --resource-group MyResourceGroup --license-type windows_server
-  - name: Update a VM instance's protection policies.
-    text: az vmss update --name MyScaleSet --resource-group MyResourceGroup --instance-id 4 --protect-from-scale-set-actions False --protect-from-scale-in
-  - name: Update a VM instance's protection policies.
-    text: az vmss update --name MyScaleSet --resource-group MyResourceGroup --instance-id 4 --set protectionPolicy.protectFromScaleIn=True protectionPolicy.protectFromScaleSetActions=False
-
```

Comment:
Where do these lines go?

Reply:
It's a bug. fixing. Seems have to add them back manually for this PR in case of creating another PR which needs more time.
I will fix them in branch Aladdin-src.

Reply:
Fix here https://github.com/haroldrandom/azure-cli/commit/6a78f7c9491d04a75da1430066456a51aa06d8e2